src/web/src/App.vue

```diff
@@ -5,44 +5,17 @@
 </template>
 
 <script>
-import { TOGGLE_DARK_MODE, SET_COURSE_LIST } from "@/store";
```

Comment:
Are these changes from merging? should maybe prefer rebasing to avoid having these commits duplicated and unexpected changes